Jamulus.pro

```diff
@@ -93,22 +93,31 @@ win32 {
 
     # replace ASIO with jack if requested
     contains(CONFIG, "jackonwindows") {
-        message(Using Jack instead of ASIO.)
-
-        !exists("C:/Program Files (x86)/Jack/includes/jack/jack.h") {
-            message(Warning: jack.h was not found at the usual place, maybe jack is not installed)
+        contains(QT_ARCH, "i386") {
+            message("Native i386 build")
+            libjackname = libjack.lib
+        } else {
+            message("Native x86_64 build")
+            libjackname = libjack64.lib
+            !exists("C:/Program Files (x86)/JACK2/include/jack/jack.h") {
+                message("Warning: jack.h was not found at the usual place, install JACK2 32 bits to cross compile.")
+            }
         }
-
-        HEADERS -= windows/sound.h
-        SOURCES -= windows/sound.cpp
-        HEADERS += linux/sound.h
-        SOURCES += linux/sound.cpp
-        DEFINES += WITH_JACK
-        DEFINES += JACK_REPLACES_ASIO
-        DEFINES += _STDINT_H # supposed to solve compilation error in systemdeps.h
-        INCLUDEPATH += "C:/Program Files (x86)/Jack/includes"
-        LIBS += "C:/Program Files (x86)/Jack/lib/libjack64.lib"
+          !exists("C:/Program Files/JACK2/include/jack/jack.h") {
+            message("Warning: jack.h was not found at the usual place, maybe JACK2 is not installed")
+          }
+
+            HEADERS -= windows/sound.h
+            SOURCES -= windows/sound.cpp
+            HEADERS += linux/sound.h
+            SOURCES += linux/sound.cpp
+            DEFINES += WITH_JACK
+            DEFINES += JACK_REPLACES_ASIO
+            DEFINES += _STDINT_H # supposed to solve compilation error in systemdeps.h
+            INCLUDEPATH += "C:/Program Files/JACK2/include"
+            LIBS += "C:/Program Files/JACK2/lib/$${libjackname}"
     }
+
```

Comment:
Just a quick note (haven't looked at the other recent updates yet): Indentation still seems to be off.

Reply:
I think it is ok in 506876e4